src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
For å muliggjøre mocking av final-klasser med mockito, da kotlin-klassene er final by default. Synes det var greit å holde seg til mockito når det enda var såpass lite kotlin i prosjektet, så kan vi heller dra inn Mockk senere hvis det blir mer?